src/pytest_docker/__init__.py

```diff
@@ -21,7 +21,11 @@ def execute(command, success_codes=(0,)):
         output = error.output or b''
         status = error.returncode
         command = error.cmd
-    output = output.decode('utf-8')
+    try:
+        output = output.decode('utf-8')
+    except UnicodeDecodeError:
+        # Decoding utf-8 from console output may fail (Windows & Python<3.6)
+        pass
```

Comment:
This can crash on trying to produce the exception message below.

Reply:
No, it won't

````
Python 3.5.4 (v3.5.4:3f56838, Aug  8 2017, 02:17:05) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> "é".encode('cp1252').decode('utf-8')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe9 in position 0: unexpected end of data
>>> "%s" % "é".encode('cp1252')
"b'\\xe9'"
>>>

Reply:
I mean output can be not set and there's code using it below.

Reply:
I think it'll at least be an empty byte array, in any case, so I don't think it can break anything below.

Reply:
Oh right, it shouldn't crash.